src/__tests__/usePopupHistory.test.ts

```diff
@@ -0,0 +1,72 @@
+import { renderHook, act } from "@testing-library/react";
+
+import { usePopup } from "../usePopup";
+import { ICalculationsRepository } from "../data/calculations/ICalculationsRepository";
+
+const mockChrome = {
+    tabs: {
+        query: jest.fn(),
+        reload: jest.fn(),
+    },
+    runtime: {
+        onMessage: {
+            addListener: jest.fn(),
+        },
+        sendMessage: jest.fn(),
+    },
+    storage: {
+        local: {
+            get: jest.fn(),
+            set: jest.fn(),
+            onChanged: {
+                addListener: jest.fn(),
+                removeListener: jest.fn(),
+            },
+        },
+    },
+};
+
+(global as any).chrome = mockChrome;
+
+describe("usePopup", () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it("refreshCalculationHistory should show all calculations", async () => {
+        const { result } = renderHook(() => usePopup());
+
+        const mockCalculationRepository = ICalculationsRepository.instance;
+        mockCalculationRepository.storeCalculation({
+            bytes: -1,
```

Comment:
how do you get -1 bytes?

Reply:
it's impossible for the user!
I just wanted to add a number that is not 0, since the default value stored in the repository is 0

Should I use some other non-negative random number instead?

Reply:
yeah I think the tests should mimic real life use cases ideally